app/directives/jqTree.directive.js

```diff
@@ -1,27 +1,14 @@
 'use strict';
 
 (function() {
-  angular.module('treeView', []).directive('tree', function($parse, SelectedFiles) {
-    var iterate_and_apply_event_recursively = function(event, node, tree) {
```

Comment:
This probably doesn't have to move; the only reason I did was to keep it in scope, since this is called by name in `on_click`. I could bind it to the tree object, however, and have `on_click` call `this.iterate_and_apply_event_recursively()`?